ny-portal/src/components/Navigation/navigation.module.scss

```diff
@@ -0,0 +1,275 @@
+@use "@fremtind/jokul/styles/core/jkl";
+
+$_sidebar-breakpoint: 1512px;
+
+.navigation-scrim {
```

Comment:
Tror ikke denne klassen brukes noe sted? Undersøkte mest fordi jeg var litt usikker på hva en scrim er for noe men ubrukt CSS er det uansett greit å bli kvitt :)